src/types/index.ts

```diff
@@ -83,12 +83,56 @@ export enum PermissionType {
   DELETE = "delete",
 }
 
+/**
+ * Represents the options for sending notifications.
+ */
+export enum SendNotification {
+  /**
+   * Send an invitation if the target user does not use the app
+   */
+  INVITE_ONLY = "invite-only",
+  /**
+   * Always send an invitation to the target user
+   */
+  ALWAYS = "always",
+  /**
+   * Never send an invitation to the target user
+   */
+  NEVER = "never",
+}
```

Comment:
Why never? Why would you create a notification for a user to never be sent? I guess I'm missing something.

And should these comments be 'send a notification' instead of 'invitation'?

Reply:
Yes, "send a notification" it should be.

You would set it to never in two cases:
* You do not want to notify the recipient in any way
* You only want to create a notification that appears on the dashboard
`sendMessage="never"` corresponds to the current `enabled=false`.